sdk/search/search-documents/test/odata.spec.ts

```diff
@@ -30,6 +30,73 @@ describe("odata", () => {
     assert.strictEqual(result, "search.ismatch('you''re', 'Description')");
   });
 
+  it("no arguments", () => {
+    assert.strictEqual(odata`Foo eq 2`, "Foo eq 2");
+  });
+
+  it("one argument", () => {
+    assert.strictEqual(odata`Foo eq ${2}`, "Foo eq 2");
+  });
+
+  it("many arguments", () => {
+    assert.strictEqual(odata`Foo eq ${2} and Bar eq ${3}`, "Foo eq 2 and Bar eq 3");
+    assert.strictEqual(
+      odata`Foo eq ${2} and Bar eq ${3} and Baz eq ${4}`,
+      "Foo eq 2 and Bar eq 3 and Baz eq 4"
+    );
+    assert.strictEqual(
+      odata`Foo eq ${2} and Bar eq ${3} and Baz eq ${4} and Qux eq ${5}`,
+      "Foo eq 2 and Bar eq 3 and Baz eq 4 and Qux eq 5"
+    );
+    assert.strictEqual(
+      odata`Foo eq ${2} and Bar eq ${3} and Baz eq ${4} and Qux eq ${5} and Quux eq ${6}`,
+      "Foo eq 2 and Bar eq 3 and Baz eq 4 and Qux eq 5 and Quux eq 6"
+    );
+  });
+
+  it("null", () => {
+    assert.strictEqual(odata`Foo eq ${null}`, "Foo eq null");
+  });
+
+  it("bool", () => {
+    let x: boolean = true;
+    assert.strictEqual(odata`Foo eq ${x}`, "Foo eq true");
+    assert.strictEqual(odata`Foo eq ${true}`, "Foo eq true");
+
+    x = false;
+    assert.strictEqual(odata`Foo eq ${x}`, "Foo eq false");
+    assert.strictEqual(odata`Foo eq ${false}`, "Foo eq false");
+  });
+
+  it("numbers", () => {
+    assert.strictEqual(odata`Foo eq ${0}`, "Foo eq 0");
+    assert.strictEqual(odata`Foo eq ${2}`, "Foo eq 2");
+    assert.strictEqual(odata`Foo eq ${-2}`, "Foo eq -2");
+    assert.strictEqual(odata`Foo eq ${2.5}`, "Foo eq 2.5");
+  });
+
+  it("limits", () => {
+    assert.strictEqual(odata`Foo eq ${Number.NEGATIVE_INFINITY}`, "Foo eq -Infinity");
+    assert.strictEqual(odata`Foo eq ${Number.POSITIVE_INFINITY}`, "Foo eq Infinity");
+    assert.strictEqual(odata`Foo eq ${Number.NaN}`, "Foo eq NaN");
+    assert.strictEqual(odata`Foo eq ${Number.MAX_VALUE}`, "Foo eq 1.7976931348623157e+308");
+    assert.strictEqual(odata`Foo eq ${Number.MIN_VALUE}`, "Foo eq 5e-324");
```

Comment:
I don't understand how this helps us test the code we wrote? Feels now like we're testing the runtime. What if a runtime defines a higher MAX_VALUE? Do we care?

Reply:
I just wanted to cover the limits case. Removed it